autoload/verilog_systemverilog.vim

```diff
@@ -586,7 +586,12 @@ function! verilog_systemverilog#VerilogErrorFormat(...)
   endif
   if (l:tool == "ncverilog")
     " Based on https://github.com/vhda/verilog_systemverilog.vim/issues/88
-    set errorformat=%.%#:\ \*%t\\,%.%#\ \(%f\\,%l\|%c\):%m
+    set errorformat=%A%p\|,%Z%.%#:\ \*%t\\,%.%#\ \(%f\\,%l\|%c\):%m
+    set errorformat+=%.%#\ *%t\\,%.%#(%f\\,%l\|%c):\ %m
+    " Assertion notices don't have column information
+    set errorformat+=%.%#\ *%t\\,%.%#(%f\\,%l):\ %m
```

Comment:
This is for the following:
ncsim: *E,ASRTST (filename.sv,769): (time 0 FS) Assertion assertionname has failed